package.json

```diff
@@ -2,11 +2,11 @@
   "name": "@aerogear/apollo-voyager-server-monorepo",
   "private": true,
   "scripts": {
-    "test": "echo ok",
-    "bootstrap": "lerna bootstrap --no-ci",
+    "test": "lerna exec -- npm test",
+    "bootstrap": "lerna bootstrap  --no-ci",
     "clean": "lerna clean --yes",
-    "lint": "tslint '*/*/src/**/*.ts' '*/*/test/**/*.ts'",
-    "format": "tslint '*/*/src/**/*.ts' '*/*/test/**/*.ts' --fix",
+    "lint": "tslint '*/*/src/**/*.ts' --exclude '*/*/src/**/*.test.ts'",
+    "format": "tslint '*/*/src/**/*.ts' --fix --force > /dev/null",
```

Comment:
Just wondering what's the `> /dev/null` thing for here? Is this going to suppress error output?

Reply:
Short answer yes:

Long answer
------------
2 scripts:
* `lint`: Complains with "ERROR: bla bla" messages, doesn't fix, exits with non-zero in case of error, only used in non-test code
* `format`: It runs on test and non-test code. Don't want to see "ERROR: bla bla" in the output because the tests might have too many of them, so that's why I put "> dev/null". I don't want it to return non-zero, so I put "--force".